File: DiplomskiAndroid/app/src/main/java/com/example/diplomskiandroid/models/Part.java
package com.example.diplomskiandroid.models;

public class Part {

    private int id;
    private String name;
    private String manufacturer;
    private double price;
    private int stockQuantity;
    private int minimumStock;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getManufacturer() {
        return manufacturer;
    }

    public double getPrice() {
        return price;
    }

    public int getStockQuantity() {
        return stockQuantity;
    }

    public int getMinimumStock() {
        return minimumStock;
    }
}

File: DiplomskiAPI/Models/Part.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DiplomskiAPI.Models
{
    [Table("parts")]
    public class Part
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("name")]
        public string Name { get; set; } = string.Empty;

        [Column("manufacturer")]
        public string? Manufacturer { get; set; }

        [Column("price")]
        public decimal Price { get; set; }

        [Column("stock_quantity")]
        public int StockQuantity { get; set; }

        [Column("minimum_stock")]
        public int MinimumStock { get; set; }

        [Column("created_at")]
        public DateTime CreatedAt { get; set; }
        public ICollection<WorkOrderPartItem> WorkOrderParts { get; set; } = new List<WorkOrderPartItem>();
    }
}